portfolio/src/main/java/com/google/sps/servlets/AnimeDataServlet.java

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/anime-data")
+public class AnimeDataServlet extends HttpServlet {
+  
+  private HashMap<String, Integer> animeVotes = new HashMap<>();
```

Comment:
This will work for now, but actual system wouldn't work this way.
1. What is this server gets rebooted?
2. What if there are multiple servers (which is typical)?
A better solution is to use some kind of a central database (e.g., blobstore) although even that has a slight issue with concurrency. You don't have to do that in this PR but letting you know so that you can think about it in your future coding.

Reply:
Yeah, I had forgotten to come back to this last week after I integrated Blobstore onto another branch. I'll update this in a different PR when I get a change, after I send in the PR for the Calendar Algorithm.